docs/guides/views.rst

```diff
@@ -295,6 +295,34 @@ If your entity list will display the entity owners, you can improve performance
 
 See also :doc:`check this page out first </design/database>`.
 
+Since 1.11, you can define an alternative view to render list items using ```'item_view'``` parameter.
```

Comment:
i'm fine with leaving 1.11 in, as 1.10 devs also read the latest docs probably, so it is good to know when to use this new feature